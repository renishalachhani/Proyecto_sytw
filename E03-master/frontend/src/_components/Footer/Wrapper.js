import styled from 'styled-components'

const Wrapper = styled.footer`
  display: flex;
  justify-content: space-between;
  padding: 3em 0;
  border-top: 1px solid #000;
  background-color: #808080;
  
`

export default Wrapper
