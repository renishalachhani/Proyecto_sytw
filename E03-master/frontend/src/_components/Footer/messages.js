/*
 * Footer Messages
 *
 * This contains all the text for the Footer component.
 */
import { defineMessages } from 'react-intl'

export const scope = 'boilerplate.components.Footer'

export default defineMessages({
  message: {
    id: `${scope}.message`,
    defaultMessage: 'This is the Footer for EventFinder!!!',
  },
  authorMessage: {
    id: `${scope}.author.message`,
    defaultMessage: `
      Made with love by {author}.
    `,
  },
})
