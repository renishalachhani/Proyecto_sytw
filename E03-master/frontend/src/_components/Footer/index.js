import React from 'react'
import { FormattedMessage } from 'react-intl'
import Wrapper from './Wrapper'
import messages from './messages'
import mail from '../../../images/correo.jpg'
import github from '../../../images/github.png'
function Footer() {
  return (
    <div className="foot">
      <div className="d-flex border-top align-items-center justify-content-around bg-light">
        <div className="bd-highlight mt-3 mb-5">
          <div className="mb-1 text-secondary">EVENTSFINDER</div>

          <div>
            This website allows you to create events, sign up for them, <br />
            join them as well as see upcoming events and <br />
            access an extensive community of events.
          </div>
        </div>

        <div className="bd-highlight mt-3 mb-5 pb-4 mr-5 pr-5">
          <div className="mb-1 text-secondary">LINK TO THE PROJECT</div>

          <div className="d-flex align-items-center">
            <img
              src={github}
              alt="logoGitHub"
              className="img-responsive img-fluid"
              height="30"
              width="30"
            />

            <a href="https://github.com/SyTW2019/E03" className="ml-1">
              Github's project
            </a>
          </div>
        </div>

        <div className="bd-highlight mt-3 mb-2">
          <div className="mb-1 text-secondary">CONTACT</div>

          <div className="d-flex align-items-center">
            <img
              src={mail}
              alt="logoCorreo"
              className="img-responsive img-fluid ml-1"
              height="15"
              width="15"
            />

            <a href="alu0101061672@ull.edu.es" className="pl-2 small">
              alu0101061672@ull.edu.es
            </a>
          </div>

          <div className="d-flex align-items-center">
            <img
              src={mail}
              alt="logoCorreo"
              className="img-responsive img-fluid ml-1"
              height="15"
              width="15"
            />

            <a href="alu0101029993@ull.edu.es" className="pl-2 small">
              alu0101029993@ull.edu.es
            </a>
          </div>

          <div className="d-flex align-items-center">
            <img
              src={mail}
              alt="logoCorreo"
              className="img-responsive img-fluid ml-1"
              height="15"
              width="15"
            />

            <a href="alu0100954609@ull.edu.es" className="pl-2 small">
              alu0100954609@ull.edu.es
            </a>
          </div>

          <div className="d-flex align-items-center">
            <img
              src={mail}
              alt="logoCorreo"
              className="img-responsive img-fluid ml-1"
              height="15"
              width="15"
            />

            <a href="alu0101028026@ull.edu.es" className="pl-2 small">
              alu0101028026@ull.edu.es
            </a>
          </div>

          <div className="d-flex align-items-center">
            <img
              src={mail}
              alt="logoCorreo"
              className="img-responsive img-fluid ml-1"
              height="15"
              width="15"
            />

            <a href="alu0101045603@ull.edu.es" className="pl-2 small">
              alu0101045603@ull.edu.es
            </a>
          </div>
        </div>
      </div>
    </div>

    // <Wrapper>

    // <section>
    //     <FormattedMessage {...messages.message} />
    // </section>

    // </Wrapper>
  )
}

export default Footer
