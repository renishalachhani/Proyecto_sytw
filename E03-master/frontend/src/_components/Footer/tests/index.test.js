import jsdom from 'jsdom'
import React from 'react'
import renderer from 'react-test-renderer'
import { mount, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import Footer from '../index'

configure({ adapter: new Adapter() })

const { JSDOM } = jsdom
const dom = new JSDOM('<!doctype html><html><body></body></html>')
const { window } = dom

global.window = window
global.document = window.document

describe('<Footer />', () => {
  test('Comprobar que se renderiza Footer', () => {
    const wrapper = mount(<Footer />)
    expect(wrapper.find(Footer)).toHaveLength(1)
  })

  it('Snapshot', () => {  
    const Component = renderer.create(<Footer />).toJSON();
    expect(Component).toMatchSnapshot();
  });
})
