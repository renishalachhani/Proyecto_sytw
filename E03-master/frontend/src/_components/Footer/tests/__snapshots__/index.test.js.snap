// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Snapshot 1`] = `
<div
  className="foot"
>
  <div
    className="d-flex border-top align-items-center justify-content-around bg-light"
  >
    <div
      className="bd-highlight mt-3 mb-5"
    >
      <div
        className="mb-1 text-secondary"
      >
        EVENTSFINDER
      </div>
      <div>
        This website allows you to create events, sign up for them, 
        <br />
        join them as well as see upcoming events and 
        <br />
        access an extensive community of events.
      </div>
    </div>
    <div
      className="bd-highlight mt-3 mb-5 pb-4 mr-5 pr-5"
    >
      <div
        className="mb-1 text-secondary"
      >
        LINK TO THE PROJECT
      </div>
      <div
        className="d-flex align-items-center"
      >
        <img
          alt="logoGitHub"
          className="img-responsive img-fluid"
          height="30"
          src="test-file-stub"
          width="30"
        />
        <a
          className="ml-1"
          href="https://github.com/SyTW2019/E03"
        >
          Github's project
        </a>
      </div>
    </div>
    <div
      className="bd-highlight mt-3 mb-2"
    >
      <div
        className="mb-1 text-secondary"
      >
        CONTACT
      </div>
      <div
        className="d-flex align-items-center"
      >
        <img
          alt="logoCorreo"
          className="img-responsive img-fluid ml-1"
          height="15"
          src="test-file-stub"
          width="15"
        />
        <a
          className="pl-2 small"
          href="alu0101061672@ull.edu.es"
        >
          alu0101061672@ull.edu.es
        </a>
      </div>
      <div
        className="d-flex align-items-center"
      >
        <img
          alt="logoCorreo"
          className="img-responsive img-fluid ml-1"
          height="15"
          src="test-file-stub"
          width="15"
        />
        <a
          className="pl-2 small"
          href="alu0101029993@ull.edu.es"
        >
          alu0101029993@ull.edu.es
        </a>
      </div>
      <div
        className="d-flex align-items-center"
      >
        <img
          alt="logoCorreo"
          className="img-responsive img-fluid ml-1"
          height="15"
          src="test-file-stub"
          width="15"
        />
        <a
          className="pl-2 small"
          href="alu0100954609@ull.edu.es"
        >
          alu0100954609@ull.edu.es
        </a>
      </div>
      <div
        className="d-flex align-items-center"
      >
        <img
          alt="logoCorreo"
          className="img-responsive img-fluid ml-1"
          height="15"
          src="test-file-stub"
          width="15"
        />
        <a
          className="pl-2 small"
          href="alu0101028026@ull.edu.es"
        >
          alu0101028026@ull.edu.es
        </a>
      </div>
      <div
        className="d-flex align-items-center"
      >
        <img
          alt="logoCorreo"
          className="img-responsive img-fluid ml-1"
          height="15"
          src="test-file-stub"
          width="15"
        />
        <a
          className="pl-2 small"
          href="alu0101045603@ull.edu.es"
        >
          alu0101045603@ull.edu.es
        </a>
      </div>
    </div>
  </div>
</div>
`;
